src/fog/Fog.unit.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+
+import { fireEvent, render, screen } from '@testing-library/react'
+
+import { Fog } from './Fog'
+
+describe('Fog', () => {
+  it('should be able to click on an interactive element', () => {
+    const onClick = jest.fn()
+    render(
+      <Fog>
+        <button onClick={onClick}>Button</button>
+      </Fog>,
+    )
+
+    const button = screen.getByRole('button')
+    fireEvent.click(button)
+    expect(onClick).toHaveBeenCalled()
+  })
+
+  it('should not be able to click on an interactive element', () => {
+    const onClick = jest.fn()
+    render(
+      <Fog isLoading>
+        <button onClick={onClick}>Button</button>
+      </Fog>,
+    )
+
+    const button = screen.getByRole('button')
+    expect(onClick).not.toHaveBeenCalled()
```

Comment:
Oops I forgot to `fireEvent.click(button)`  that's why the is passing 🤦‍♂️

Reply:
I updated the test in my last commit, since I'm using `aria-hidden`, the button shouldn't even be "readable"